acceptance/tests/config/puppet_manages_own_configuration_in_robust_manner.rb

```diff
@@ -5,8 +5,6 @@
 # expect that after correcting their actions, puppet will work correctly.
 test_name "Puppet manages its own configuration in a robust manner"
 
-confine :except, :platform => 'fedora-19'
```

Comment:
why?  is there a ticket for this?
should we use skip_test with a comment, where possible?
should we use expect_failure when we expect one of the asserts to throw an exception?